.github/workflows/deploy_switch_ensdb_playground_environment.yml

```
@@ -0,0 +1,60 @@
+          esac
+
+      - name: Switch Traefik Routing
+        run: |
+          sudo apt update
+          sudo apt install -y redis-tools
+
+          # ENSDB PLAYGROUND
+          redis-cli -u $REDIS_URL SET traefik/http/routers/ensdb-playground-router/service "${TARGET_ENVIRONMENT}-ensdb-playground"
+
+      - name: Send Slack Notification
+        uses: ./.github/actions/send_slack_notification
```

actionlint (1.7.12)

[error] description is required in metadata of "Send Slack notification" action at "/home/jailuser/git/.github/actions/send_slack_notification/action.yml"

(action)